Sources/Imperial/Routing/FederatedServiceRouter.swift

```diff
@@ -76,4 +76,19 @@ extension FederatedServiceRouter {
             }
         }
     }
+
+    public func upgradeRelativeCallbackURL(from request: Request) {
```

Comment:
I'm not sure I see the whole benefit of adding this in. Since callbacks are generally set in environment variables, it makes it easy to switch that out. In addition to the breaking change requirement above, I don't think this should be included. What do you think @calebkleveter ?

Reply:
I vaguely recall having discussed this on Discord, but cannot find the discussion anymore. This is not a make or break feature for me, though I find it very useful in my code (and it would not be that easy to retrofit in an `extension`, I think.

Reply:
Cool, I'll leave for @calebkleveter to make a call on then

Reply:
Looking at this, I think we should leave this specific feature out of the PR. Since it is a breaking change, we can open an issue so we can add it to Imperial when I do the re-write (someday within the next millennia).

Reply:
OK, I'll. revise.